content/saasquatch-api.yaml

```diff
@@ -1556,28 +1550,31 @@ paths:
         200:
           description: Referral code found
           examples:
-          - code: "BOBTESTERSON"
-            dateCreated: 1411600868797
-            referrerName: "Bob T"
-            reward:
-              $ref: "#/definitions/RewardTemplate/example/CREDIT"
-          - code: "BOBTESTERSON"
-            dateCreated: 1411600868797
-            referrerName: "Bob T"
-            reward:
-              $ref: "#/definitions/RewardTemplate/example/PCT_DISCOUNT"
+          "application/json":
+            - code: "BOBTESTERSON"
+              dateCreated: 1411600868797
+              referrerName: "Bob T"
+              programId: "referral-a"
+              reward:
+                $ref: "#/definitions/RewardTemplate/example/CREDIT"
+            # - code: "BOBTESTERSON"
+            #   dateCreated: 1411600868797
+            #   referrerName: "Bob T"
+            #   programId: "referral-b"
+            #   reward:
+            #     $ref: "#/definitions/RewardTemplate/example/PCT_DISCOUNT"
           schema:
             $ref: "#/definitions/Code"
         403:
-            description: Endpoint disabled
-            schema:
-              $ref: "#/definitions/Error"
+          description: Endpoint disabled
+          schema:
+            $ref: "#/definitions/Error"
         404:
           description: Not Found
           schema:
             $ref: "#/definitions/Error"
-  "/{tenant_alias}/open/code/{referral_code}/account/{account_id}/user/{user_id}":
-    post:
+  ? "/{tenant_alias}/open/code/{referral_code}/account/{account_id}/user/{user_id}"
+  : post:
```

Comment:
What is the `?` and `:` here?

Reply:
weird, good catch I think prettier did that